cob_map_accessibility_analysis/CMakeLists.txt

```diff
@@ -1,28 +1,20 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_map_accessibility_analysis)
 
-set(catkin_RUN_PACKAGES
```

Comment:
Why did you remove this handy trick to avoid listing all dependencies twice in this file? Is there any good reason why we should do this e.g. in the vision packages as well or is it just preferred style?

Reply:
like this, the packages are not alphabetically sorted - i.e. `message_generation`/`message_runtime` - which is required by `catkin_lint`

Reply:
Ah, I see. Thanks for the explanantion.